controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -343,59 +343,60 @@ public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenChec
         };
 
         WireCommands.CreateTransaction request = new WireCommands.CreateTransaction(idGenerator.get(),
-                Segment.getScopedName(scope, stream, segmentNumber), txId, delegationToken);
+                qualifiedName, txId, delegationToken);
         sendRequestAsync(request, replyProcessor, result, clientCF, ModelHelper.encode(uri));
         return result;
     }
 
     public CompletableFuture<TxnStatus> commitTransaction(final String scope,
                                                           final String stream,
-                                                          final int segmentNumber,
+                                                          final long segmentId,
                                                           final UUID txId,
```

Comment:
All over this file (and in a few others) txId is still here. Why? Isn't the segmentId now containing that information?

Reply:
No, transaction ids will continue to be there. answered here
https://github.com/pravega/pravega/pull/2600#issuecomment-391576540